volume/parser_lcow_impl.go

```diff
@@ -0,0 +1,35 @@
+package volume
```

Comment:
The thing is that for future-proof multi-arch/os system, this PR makes all flavors of the parser available on all OS. I also wanted file ordering make different parsers appear together (so they all have a `parser_` prefix. However, if I chose to keep names as `parser_linux.go` / `parser_windows.go` it would have triggered conditional compilation, so I had to add a new suffix for avoiding it to happen.

Reply:
Another workaround is `-` instead of `_` but that is inconsistent with most of the file names in the project. Yet another is to modify the platform name so it will not match exactly, but I can't come up with a satisfactory modification.

If not for the file ordering issue, which seems minor to me, I would name them `lcow_parser.go`, `linux_parser.go`, and `windows_parser.go`, since that is closer to the respective type names.

Reply:
@johnstep @jhowardmsft just did that. With the small number of files in this package, I suppose it is ok to have them not grouped together.